tests/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+
+services:
+  dev-pihole:
+    image: ghcr.io/pi-hole/pihole:latest
+    volumes:
+      - ../:/var/www/html/admin:ro
+    environment:
+      WEBPASSWORD: admin
+
+  npm:
+    image: mcr.microsoft.com/playwright:v1.28.1
```

Comment:
using Playwright's official image as the Node/NPM image ensures versions are always compatible - and one less image to download